src/code/z_actor.c

```diff
@@ -852,7 +852,24 @@ void Actor_FreeOverlay(ActorOverlay* entry) {
 
 #pragma GLOBAL_ASM("./asm/non_matchings/code/z_actor//func_800BE03C.asm")
 
-#pragma GLOBAL_ASM("./asm/non_matchings/code/z_actor//func_800BE0B8.asm")
+Actor *func_800BE0B8(GlobalContext *globalCtx, Actor *inActor, s16 arg2, u8 arg3, f32 arg4) {
```

Comment:
As we said in Discord, z_actor is reserved. I guess we can pass it this time, but for the next time you could at least ask if it is OK matching it.
Adding functions prototypes is OK btw.